includes/Generator/Coupon.php

```diff
@@ -24,13 +24,16 @@ class Coupon extends Generator {
 	public static function generate( $save = true, $assoc_args = array() ) {
 		parent::maybe_initialize_generators();
 
-		$defaults = array(
-			'min' => 5,
-			'max' => 100,
-		);
+	$defaults = array(
+		'min'           => 5,
+		'max'           => 100,
+		'discount_type' => 'fixed_cart',
+	);
```

Comment:
Nit: The indentation is not consistent.

```suggestion
		$defaults = array(
			'min'           => 5,
			'max'           => 100,
			'discount_type' => 'fixed_cart',
		);
```